components/ruby/lib/license_acceptance/strategy/argument.rb

```diff
@@ -6,39 +6,39 @@ module Strategy
     # Look for acceptance values in the ARGV
     class Argument < Base
 
+      FLAG = "--chef-license".freeze
```

Comment:
Thanks for cleaning this file up! Its easier to read and the tests are more helpful 💥